fix-panels.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+# Script sets correct names for plugins folders.
+
+import json
+import os
+import shutil
+import sys
+
+GRAFANA_PLUGINS_DR = '/var/lib/grafana/plugins/'
+
+
+def rename_panels():
+    for app in ['panels']:
+        print '  * Renaming %r' % (app,)
+        panels_list = os.listdir(GRAFANA_PLUGINS_DR)
+        for panel in panels_list:
+            panel_path = os.path.join(GRAFANA_PLUGINS_DR, panel, 'dist/plugin.json')
+            if os.path.exists(panel_path):
+                print '   * %r -> ' % (panel,),
+                with open(panel_path, 'r') as f:
+                    panel_params = json.loads(f.read())
+                    print '%r' % (panel_params['id'],)
+                    if (not(os.path.exists(os.path.join(GRAFANA_PLUGINS_DR, panel_params['id'])))):
+                        print '    * Renaming panel'
+                        shutil.copytree(os.path.join(GRAFANA_PLUGINS_DR, panel), os.path.join(GRAFANA_PLUGINS_DR, panel_params['id']))
+                    if (panel != panel_params['id']):
+                        print '    * Removing leftovers'
```

Comment:
I'd suggest to call it `Cleaning up...` or something similar, since it's not clear why it a panel is a leftover.